lldb/source/DataFormatters/TypeSummary.cpp

```diff
@@ -145,15 +148,34 @@ std::string CXXFunctionSummaryFormat::GetDescription() {
   return std::string(sstr.GetString());
 }
 
+ConstString CXXFunctionSummaryFormat::GetName() {
+  return ConstString(m_description);
+}
+
+ConstString CXXFunctionSummaryFormat::GetImplType() {
+  return ConstString("c++");
+}
+
```

Comment:
We should switch to `std::string` instead of `ConstString.. These objects aren't starting out as ConstString objects. We only want to use ConstString objects for things we are searching for by name where there will be many copies of the string.